config/config.go

```diff
@@ -54,6 +54,11 @@ type Subscriber struct {
 	Type  string `default:"mock"`
 	Redis Redis
 	Mock  Mock
+	API   API
+}
+
+type API struct {
```

Comment:
__[Golint]__

exported type API should have comment or be unexported


<!-- Comment by [SideCI](https://sideci.com) -->